src/components/Homepage/events/getEvents.js

```diff
@@ -2,6 +2,7 @@ import { format, isAfter } from 'date-fns'
 
 export const getHomepageMeetups = events =>
   events
+    .sort((a, b) => (a.node.date < b.node.date ? -1 : 1))
```

Comment:
what's the var type of the `node.date` property? is it a string or number? if a number, I agree with this change, but if it's a string you'd be making a string comparison which would give you wrong results.

I'm seeing the line 7 we're doing `new Date(n.node.date)` so I'm just making sure we're comparing the correct types.

Finally, and this is a small improvement, we should sort after the filtering, so instead of sorting and reducing the collection size, we can only sort on the necessary items :)

Reply:
That's true. Let me double-check and update if needed.